src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
That's something we can refactor to later, now the interface is there.

`MagicAwareAccessInspector` makes sense to me, you sold me on it.